x-pack/solutions/observability/plugins/apm_data_access/server/plugin.ts

```diff
@@ -51,8 +51,8 @@ export class ApmDataAccessPlugin
   }
 
   public start(core: CoreStart, plugins: ApmDataAccessServerDependencies) {
-    const getApmIndicesWithInternalUserFn = () => {
-      const soClient = core.savedObjects.createInternalRepository();
+    const getApmIndicesWithInternalUserFn = (request: KibanaRequest) => {
+      const soClient = core.savedObjects.getScopedClient(request);
       return plugins.apmSourcesAccess.getApmIndices(soClient);
     };
```

Comment:
@bryce-b  is fixed this already here https://github.com/elastic/kibana/pull/214002

Reply:
Yes, this is here because it got caught up in a merge conflict.